src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs

```diff
@@ -128,6 +128,14 @@ public class TestConnectionCommand : PSCmdlet
         public Int32 BufferSize { get; set; } = 32;
 
         /// <summary>
+        /// The following is the definition of the input parameter "TimeOut".
+        /// Time-out value in milliseconds. If a response is not received in this time, no response is assumed. The default is 1000 milliseconds.
+        /// </summary>
+        [Parameter]
+        [ValidateRange((int)1, Int32.MaxValue)]
```

Comment:
I've never liked this use of `ValidateRange` - it seems like we should have `ValidatePositive` and `ValidateNonNegative` attributes.